resources/test/fixtures/C417.py

```diff
@@ -0,0 +1,6 @@
+nums = [1, 2, 3]
+map(lambda x: x + 1, nums)
+map(lambda x: str(x), nums)
+list(map(lambda x: x * 2, nums))
+set(map(lambda x: x % 2 == 0, nums))
+dict(map(lambda v: (v, v**2), nums))
```

Comment:
```
cargo run resources/test/fixtures/C417.py --select C417
```

gives:

```
resources/test/fixtures/C417.py:2:1: C417 Unnecessary map usage - rewrite using a generator expression
resources/test/fixtures/C417.py:3:1: C417 Unnecessary map usage - rewrite using a generator expression
resources/test/fixtures/C417.py:4:1: C417 Unnecessary map usage - rewrite using a list comprehension
resources/test/fixtures/C417.py:4:6: C417 Unnecessary map usage - rewrite using a generator expression
resources/test/fixtures/C417.py:5:1: C417 Unnecessary map usage - rewrite using a set comprehension
resources/test/fixtures/C417.py:5:5: C417 Unnecessary map usage - rewrite using a generator expression
resources/test/fixtures/C417.py:6:1: C417 Unnecessary map usage - rewrite using a dict comprehension
resources/test/fixtures/C417.py:6:6: C417 Unnecessary map usage - rewrite using a generator expression
```

Reply:
`flake8-comprehensions` creates a set and stores already-visited map calls to ignore them. 

https://github.com/adamchainz/flake8-comprehensions/blob/baa8acd66b4582e473d21143ddd3f76887ab9c39/src/flake8_comprehensions/__init__.py#L49

Reply:
I think it's ok to have multiple checks on one line even if slightly redundant. (At least, I prefer that to having to track already-visited map calls.)